kodilla-stream/src/main/java/com/kodilla/optional/homework/Student.java

```diff
@@ -0,0 +1,22 @@
+package com.kodilla.optional.homework;
+
+import java.util.Optional;
+
+public class Student {
+    String name;
+    Teacher teacher;
+
+    public Student(String name, Teacher teacher) {
+        this.name = name;
+        this.teacher = teacher;
+    }
+
+    public String getName() {
+        return name;
+    }
+
+    public Teacher getTeacher() {
```

Comment:
Ta metoda miala zwracac optional -zrobiles to samo co bylo, tylko w innym miejscu.